TypeCobol.Analysis/Graph/CfgDotFileForNodeGenerator.cs

```diff
@@ -144,6 +132,7 @@ protected override bool EmitBasicBlock(BasicBlock<Node, D> block, int incomingEd
                 {
                     sw.WriteLine(string.Format("Block{0} -> \"\" [style=dashed]", block.Index));
```

Comment:
@fm-117 So as we are always using explicit iterative perform procedure style, we can display a plain arrow style. This can be detected using IsExplicitIterativeGroup property on the group.

Reply:
Actually this is not used. The flag `GroupGrafted` is used in Extended mode and we keep the dashed style in Normal mode so I've removed property `IsExplicitIterativeGroup`. Dot files stay the same and are in-sync with those from [1260_TypeCobolCFG](https://github.com/TypeCobolTeam/TypeCobol/tree/1260_TypeCobolCFG).